token/src/test.rs

```diff
@@ -21,16 +23,85 @@ fn create_token<'a>(e: &Env, admin: &Address) -> TokenClient<'a> {
     TokenClient::new(e, &token_contract)
 }
 
+mod non_mux_client {
+    soroban_sdk::contractimport!(
+        file = "target/wasm32-unknown-unknown/release/soroban_token_contract.wasm"
+    );
+}
+
+mod mux_client {
+    soroban_sdk::contractimport!(
+        file = "target/wasm32-unknown-unknown/release/soroban_token_contract.wasm",
+        expose_muxed_addresses = true
+    );
+}
```

Comment:
I think the fact that every contract has two APIs, is a reasonably complex narrative.

Reply:
This is just an option, we don't have to go this way. I just think it works fine for hiding unnecessary details 99% of the time and provides the flexibility 1% of the time this is actually necessary, similar to some other trickier concepts like non-rooted auth. But with a separate function I agree there is not much point in doing this.